src/main/java/graphql/execution/instrumentation/dataloader/IntMap.java

```diff
@@ -0,0 +1,62 @@
+package graphql.execution.instrumentation.dataloader;
+
+import graphql.Internal;
+
+import java.util.Arrays;
+
+@Internal
+public class IntMap {
```

Comment:
Naming is hard - maybe called it LevelMap since its tracks values at specific levels